source/includes/api/responses/progress.json

```diff
@@ -15,6 +15,23 @@
             {
                "Source":"cluster0: localhost:27017",
                "Destination":"cluster1: localhost:27018"
+            },
+         "verification":
```

Comment:
We will be adding a few new fields to the /progress endpoint as part of https://jira.mongodb.org/browse/REP-5164. 
We will add `SrcTotalDocumentCount`, `DstTotalDocumentCount` to the top-level response and `HashedDocumentCount` to the `VerificationProgress`.

Reply:
New fields added.